### PR TITLE
[PWA] Stricter linting

### PR DESCRIPTION
**Pull request by fangeugene**

### CODE REVIEW

File: pwa/app/components/ui/carousel.tsx [hunk 1]
```diff
@@ -124,8 +124,7 @@ const Carousel = React.forwardRef<
           carouselRef,
           api: api,
           opts,
-          orientation:
-            orientation || (opts?.axis === 'y' ? 'vertical' : 'horizontal'),
+          orientation,
```
Comment (fangeugene) [inline pwa/app/components/ui/carousel.tsx:127]: This seems correct from a typing perspective (`orientation` seems to never be null), but it still spooks me that the original component had this extra logic @tervay

**Comment by tervay** [comment]
requesting https://www.npmjs.com/package/eslint-plugin-unused-imports

**Comment by tervay** [comment]
the eslint plugin ecosystem (and configs) has got to have some of the absolute worst naming schemes i've ever seen

**Review by tervay — APPROVED** [review]

**Comment by fangeugene** [comment]
> requesting https://www.npmjs.com/package/eslint-plugin-unused-imports

Why is this necessary? don't we already check for this?

image

**Comment by tervay** [comment]
> > requesting [npmjs.com/package/eslint-plugin-unused-imports](https://www.npmjs.com/package/eslint-plugin-unused-imports)
> 
> Why is this necessary? don't we already check for this?
> 
> image

oops, yes. missed this

**Pull request merged**